lib/global_registry_bindings/entity/push_entity_methods.rb

```diff
@@ -35,6 +40,7 @@ def create_entity_in_global_registry
                                                                                global_registry_entity.type)
           model.update_column(global_registry_entity.id_column, # rubocop:disable Rails/SkipsModelValidations
                               global_registry_entity.id_value)
+          update_fingerprint if global_registry_entity.fingerprint_column.present?
```

Comment:
I think you should move `if global_registry_entity.fingerprint_column.present?` to `update_fingerprint`.